code/modules/client/preferences.dm

```diff
@@ -228,7 +228,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 
 			if(HAIR in pref_species.species_traits)
 
-				dat += "<td valign='top' width='21%'>"
+				dat += "<td valign='top' width='10%'>"

```

Comment:
All these `%` changes change how this stuff ends up looking in the character setup menu. The more width is has the larger it is, duh. Some of the pre-existing stuff had width values far too large for the content they had, making the new stuff have to squish up in a corner. Not good.